rust-engine/src/printer/pretty_ast.rs

```diff
@@ -74,7 +77,18 @@ impl<T: serde::Serialize> Display for DebugJSON<T> {
             Ok(count)
         }
 
-        const PATH: &str = "/tmp/hax-ast-debug.json";
+        /// Drop the file at `PATH` when we first write
+        fn cleanup() {
```

Comment:
Maybe something like this?

```suggestion
        fn clean_up() {
```